docs/xml-config-file.rst

```diff
@@ -1,8 +1,9 @@
 Integration Services XML Configuration
 ======================================
 
-Integration Services (IS) uses a XML configuration file to create its connectors. This XML file can contains the
-the following sections, all inside a root <is> label.
+Integration Services (IS) uses a XML configuration file to create its connectors.
+
+This XML file can contain the following sections, all inside a root <is> label.
```

Comment:
Bellow you talk about xml types. If there is any link to fastrtps documentation, put it. In other case, note to do it in future when this document will be written.

Reply:
Added.